libs/langchain/langchain/callbacks/openai_info.py

```diff
@@ -31,8 +31,19 @@
     "gpt-3.5-turbo-0613-completion": 0.002,
     "gpt-3.5-turbo-16k-completion": 0.004,
     "gpt-3.5-turbo-16k-0613-completion": 0.004,
-    # Others
+    # Azure GPT-35 input
     "gpt-35-turbo": 0.002,  # Azure OpenAI version of ChatGPT
+    "gpt-35-turbo-0301": 0.002,  # Azure OpenAI version of ChatGPT
```

Comment:
seems like all of the prompt tokens for gpt-35-turbo-... are $0.0015

https://techcommunity.microsoft.com/t5/azure-ai-services-blog/introducing-new-and-updated-models-to-azure-openai-service/ba-p/3860351

don't know if there's a more official price page for these

Reply:
Official docs still say GPT-3.5-Turbo is at 0.002.

And the way I understand the article is that _new_ versions of the model - 0613 are splitting pricing between prompt and completion tokens. However the version 0301 is still at 0.002.

I will try to find out though.

Reply:
i think there's a difference between gpt-3.5-turbo and gpt-35-turbo but could be misunderstanding (docs are pretty confusing on this)

Reply:
So Microsoft is naming all their instances gpt-35-turbo (no dot). And the same thing is coming from LLM response.
![image](https://github.com/langchain-ai/langchain/assets/865473/9f2d9b72-89f9-41dc-b956-eda89ebf6c09)

Reply:
right, which is what makes me suspect this is the right price
<img width="759" alt="Screenshot 2023-08-03 at 12 01 01 AM" src="https://github.com/langchain-ai/langchain/assets/22008038/3635629a-cf1f-4e7f-85da-74527c2e96c8">

Reply:
and it seems like (at least from I can see in the portal) it's not possible to choose or change the version to 0613, so it seems to be fair to assume it's the way you described it. I'm gonna ammend the pricing mapping.

Reply:
@baskaryan I have updated the pricing to 0.0015 for prompt tokens for all gpt-35-turbo instances.